builder/builder-next/builder.go

```diff
@@ -32,7 +32,10 @@ import (
 	grpcmetadata "google.golang.org/grpc/metadata"
 )
 
-var errMultipleFilterValues = errors.New("filters expect only one value")
+var (
+	errMultipleFilterValues = errors.New("filters expect only one value")
+	errSynonymFilterValues = errors.New("synonym filters are forbidden")
```

Comment:
I think most similar errors, we use `"conflicting ....."` (e.g. https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/runconfig/errors.go#L13-L17)


Perhaps `conflicting filters: "until" and "unused-for"` ?